### ISSUE TITLE
[v4] a case of `\middle /` rendering incorrectly with mathjax-stix2

### ISSUE THREAD

**Issue by pkra** [mathjax/MathJax#3560]
I found an odd bug that seems specific to mathjax's stix2. 

The example is:

```tex
\begin{equation*} \mathcal{H}^{2} := \left.\Set{ (X,p,\mathbf{v}) : \begin{array}{c} X {\text{ an oriented hyperbolic surface,}} \\p \in X, \mathbf{v} \in T^1_p X \end{array}} \middle / \begin{array}{c} {\text{orientation- and base- }} \\{\text{vector-preserving isometry}} \end{array}\right. \end{equation*}
```

With new-cm, this renders as follows (on mathjax.org#demo)

<img width="1336" height="101" alt="Image" src="https://github.com/user-attachments/assets/78f605b9-9ef9-404a-8110-ee65c0395be6" />

This matches xelatex with CM:

<img width="623" height="56" alt="Image" src="https://github.com/user-attachments/assets/81f3c446-c2d0-4c51-b755-d6c896fc3e0a" />

Similarly, xelatex looks like this with STIX Two:

<img width="896" height="77" alt="Image" src="https://github.com/user-attachments/assets/a2acf596-1cdd-4379-9232-83cef0e6e805" />

but MathJax v4 (tested with v4.0.0 and v4.1.2)

<img width="986" height="129" alt="Image" src="https://github.com/user-attachments/assets/f027fe01-52f2-46cc-ad3e-8d4311715570" />

**Comment by dpvc** [comment]
Looks like there `/` is STIX2 isn't symmetric about the math axis, as it is in some of the other fonts. `\middle` probably should include `symmetric="true"` on the `mo` it generates.  Do you need a patch for this?

**Comment by pkra** [comment]
Thanks for looking into it, Davide. It's not urgent from our end.

**Comment by pkra** [comment]
I can confirm that this is fixed for our use case using v4.1.3.

**Comment by dpvc** [comment]
Thanks for the confirmation!
